model/project.go

```diff
@@ -1769,14 +1769,14 @@ func (p *Project) IsGenerateTask(taskName string) bool {
 	return ok
 }
 
-func (p *Project) findAliasesForPatch(alias string, patchDoc *patch.Patch) ([]ProjectAlias, error) {
-	vars, shouldExit, err := FindAliasInProjectOrRepoFromDb(p.Identifier, alias)
+func FindAliasesForPatch(identifier, alias string, patchDoc *patch.Patch) ([]ProjectAlias, error) {
```

Comment:
nit: can we change identifier to projectId ? (Unfortunately patch identifier points to project ID instead of project identifier since those didn't use to be distinct but using identifier here makes it unclear that this isn't project identifier)